docs/admin/install.md

```diff
@@ -8,6 +8,30 @@
 !!! note
     Please check the [dedicated page](compatibility_matrix.md) for a full compatibility matrix and the deprecation policy.
 
+### Potential Apps Conflicts
+
+!!! warning
```

Comment:
Leave this warning in the prerequisites section and link to the upgrade page. It'll be good for visibility. 

As all the instructions in this subsection are for upgrading, it should move to the upgrade guide.

Reply:
Fixed